build/buildRunTimeImageBases.sh

```diff
@@ -52,13 +52,6 @@ docker build \
     -t "$RUNTIME_BASE_IMAGE_NAME-stretch" \
     $REPO_DIR
 
-docker build \
```

Comment:
Do you know if we'll be able to switch all new minor versions updates to Bullseye moving forward? For example, as Antares integrate with a specific version of Buster/Bullseye, do you know if it might be OK to move from one to the other for a `patch` update, let's say?

Reply:
Also, I see below that in some cases we kept the Buster logic. Could you please clarify in which case we kept it?

Reply:
Yes, I think it would not be hard for us to achieve that. If Antares ask us to update a specific minor or patch versions for these bullseye images, we can update it easily. From now on, we'll have all minor and patch versions of node 14, 16 and python 3.7 images to be bullseye based.

Reply:
OK thanks! Just to be 100% sure, as I believe this file is shared by all languages: if Antares ask us for an update on a version that we still provide as Buster, would we be able to do that without transitioning this image to Bullseye?
If not, then we might want to check with them if we need to keep the capacity to do that, as I believe that going from Buster to Bullseye for a minor/patch version might be tricky for them.

Reply:
Thanks for pointing out, I think from now on we'll keep these minor and patch versions consistent to be bullseye based, it makes more sense for both Antares team and us to have all images to be bullseye based, not partially buster and partially bullseye. I imagine that having different debian flavor of images for node 14 and 16 could be confusing.

Reply:
I agree with you, and my concern is for languages that we didn't move to Bullseye yet. For example, Python 3.9 is built on Buster. Antares is using the Buster based image in their codebase. If they need us to support a new minor or patch version on Buster for Python 3.9, will we be able to do it? Or will we be have to use Bullseye for it, and then create the Buster/Bullseye discrepancy we were talking about above?

Reply:
Yes we're able to achieve that, for python 3.9 we're still able to update minor or patch versions based on the buster image. They won't be impacted by this changes.

Reply:
Great, thanks for confirming!